sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -473,11 +473,11 @@ export interface CustomFormModel {
   /**
    * Date and time (UTC) when the custom model training request was received.
    */
-  requestedOn: Date;
```

Comment:
should we update the docstring for trainingStartedOn to something like "Date and time (UTC) when the custom model training started"

Reply:
Yes good point!